### PR TITLE
Adds tls connect support to client

### PR DESCRIPTION
**Pull request by mitjaziv**
This change adds secure tls connect to mqtt client.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -79,6 +82,11 @@ func main() {
 		log.Fatalf("Invalid arguments: messages count should be > 1, given: %v", *count)
 	}
```
Comment (krylovsk) [inline main.go:84]: Let's add validation that either both `cert` and `key` must be provided, or none of them?

**Review by krylovsk — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (mitjaziv) [reply main.go:84]: @krylovsk I added command line parameter validation, let me know if this is how you imagined it.

Let me know if you want me to squash this in to one commit.

**Comment by mitjaziv** [comment]
Forced push to resolve merge conflicts.

File: main.go (hunk 1 above)
Comment (krylovsk) [reply main.go:85]: This looks good, thanks.

**Pull request merged**